Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -83,8 +89,13 @@ extension BrowserViewController: TabManagerDelegate {
 
         previous?.webView.navigationDelegate = nil
         selected?.webView.navigationDelegate = self
+        
+        previous?.delegate = nil
+        selected?.delegate = self
```

Comment:
Since the WebView (and now Browser) pass themselves along in their delegates, maybe we should just leave these delegates attached all the time? When they're called, we can just bail if webView != selectedTab.webView (or browser != selectedTab).